etc/dnf/plugins/snapper.conf

```diff
@@ -0,0 +1,2 @@
+[main]
```

Comment:
I prefer to add  `enabled = 1` to the default configuration file to clarify how the plugin can be disabled.